ocaml/database/db_upgrade.ml

```diff
@@ -12,9 +12,7 @@
  * GNU Lesser General Public License for more details.
  *)
 
-module D = Debug.Make (struct
-  let name = "xapi" (* this is set to 'xapi' deliberately! :) *)
```

Comment:
And now it has been changed away from 'xapi' deliberately? :)

Reply:
I spelunked and found no justification for it in the commit that introduced the module. It's more helpful to have the actual module that prints the loglines, same reasoning I used to change all the unjustified ones 3 years back, it just helps whith triaging